src/components/DevelopmentServerPane/DevelopmentServerPane.js

```diff
@@ -177,11 +177,10 @@ const mapStateToProps = state => {
 
   return {
     project: selectedProject,
-    task: getDevServerTaskForProjectId(
-      state,
-      selectedProject.id,
-      selectedProject.type
-    ),
+    task: getDevServerTaskForProjectId(state, {
+      projectId: selectedProject.id,
+      projectType: selectedProject.type,
+    }),
```

Comment:
I updated selectors with arguments to put their additional arguments in a `props` object. This is how Reselect requires that it be done, to consider the arguments in memoization when using React props